lib/config.js

```diff
@@ -80,6 +91,7 @@ export default function subscribeConfig(conn, configChanged) {
     initPromise = Promise.all([
       conn.subscribeEvents(processComponentLoaded, 'component_loaded'),
       conn.subscribeEvents(processServiceRegistered, 'service_registered'),
+      conn.subscribeEvents(processServiceRemoved, 'service_removed'),
```

Comment:
This will return an unsub function that you should destructure in line 99 and call after line 103.